app/controllers/course/achievement/condition/achievements_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Course::Achievement::Condition::AchievementsController <
+  Course::Condition::AchievementsController
+  include Course::AchievementConditionalConcern
```

Comment:
Yeah, but we need to write the same `set_conditional` in all the condition controllers for a specific conditional. In this case, we need to write for both `Course::Achievement::Condition::AchievementsController` and `Course::Achievement::Condition::LevelsController`.